lib/reducers/rootReducer.js

```diff
@@ -947,6 +945,78 @@ const editorReducer = (state = defaultData, action) => {
         }
       };
 
+    case actionTypes.LOAD_DUMMY_DASHBOARDS:
+      return {
+        ...state,
+        dashboardList: [
```

Comment:
Could you please just for better readability move default dashboard fixtures to separate file and just import it ?